modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1116,6 +1116,92 @@ definition {
 		}
 	}
 
+	@description = "Verify if actions works with oldValue function when trigger is On Payment Status Update (Notification) with order status field"
+	@priority = 4
+	test CanCreateActionNotificationUsingOldValueWithOnPaymentStatusUpdateTrigger {
```

Comment:
@timpak Since the test is to cover if the oldValue function works on the Commerce Order object, are not important we explicit at least the oldValue function on the testcase name?